frontend/src/components/Forms/Fields/index.tsx

```diff
@@ -221,6 +221,7 @@ export const FormInput = ({
             as={AutocompleteTable}
             onChange={(newValue: string) => {
               setFieldValue?.(fieldName, newValue);
+              customOnChange(newValue);
```

Comment:
really, this is it!

Reply:
yes lol sometimes a a major problems solution is one line